### ISSUE TITLE
Interpretation result flow2rgb fonction 

### ISSUE THREAD

**Issue by chenyuZha** [hzwer/arXiv2020-RIFE#231]
你好， 我在查看`flow2rgb` 结果的时候，想请教一些问题:

- 一开始生成的flow在 `warp` fonction里，我注意到你做了一个变换: 
```
    tenFlow = torch.cat([tenFlow[:, 0:1, :, :] / ((tenInput.shape[3] - 1.0) / 2.0),
                         tenFlow[:, 1:2, :, :] / ((tenInput.shape[2] - 1.0) / 2.0)], 1)
```
想请教一下这个变换的目的是? 因为在codes里visualize的flow是在这个变换之前的。

- 用来visualize的flow是 `flow[-1][:2]`,  所以是取了最后一个scale(就是scale=1)以及对应的应该是是warp_img0的flow(t--> 1). 所以这个flow如果要找对应的image， 应该是warp_img0而不是merged? 因为我看了一下merged是warp_img0 和 warp_img1的一个fusion。
- 我查看了生成的flow以及对应每一个warp_img0。因为我选的是exp=5， 所以一共有31张。我发现flow十分的不稳定，甚至有些时候图片是空白的。想请教一下怎么去解释flow图片里的不同的颜色以及有时候是空白的?

下图是flow2rgb之后生成的31张flow:

https://user-images.githubusercontent.com/29950360/149129057-2630acf7-8796-47c4-942e-f570e6f78717.mp4

**Comment by hzwer** [comment]
这个变换看起来是试图把光流 normalize 一下；warped img0 应该和 merged 比较近了；光流小的时候颜色会变淡；exp=5 时，光流在插值过程中有大有小，所以看起来是闪烁的

**Comment by chenyuZha** [comment]
感谢解答!所以从上图来看的话， 图片的边缘是有一些artefacts ?

**Comment by hzwer** [comment]
参考TOflow的说法 光流并不是用来插帧的完美选择 如果让网络没有约束地学 它学出来的东西在外观上和光流会有些差别；特别是因为插帧中要处理物体运动的不确定性 往往无法出边缘锐利的光流
